### ISSUE TITLE
Another false negative for E122

### ISSUE THREAD

**Issue by myint** [PyCQA/pycodestyle#208]
This may be related to #201.

I expect E122 to be detected for the "Development Status" line:

``` python
setup('',
      scripts=[''],
      classifiers=[
      'Development Status :: 4 - Beta',
          'Environment :: Console',
          'Intended Audience :: Developers',
      ])
```
